### PR TITLE
Appdata for viking 1 and 2

### PR DESCRIPTION
**Pull request by scsides**
Move Viking 1 and 2 files under revision control of main ISIS git repo.

## Description
Move Viking 1 and 2 data files and env variable changes.

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [X] New feature (non-breaking change which adds functionality)
- [ ] Documentation change (update to the documentation; no code change)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [X] I have read and agree to abide by the [Code of Conduct](https://usgs-astrogeology.github.io/code/)
- [X] I have read the [**CONTRIBUTING**](https://github.com/USGS-Astrogeology/ISIS3/blob/dev/CONTRIBUTING.md) document.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

## Licensing
This project is mostly composed of free and unencumbered software released into the public domain, and we are unlikely to accept contributions that are not also released into the public domain. Somewhere near the top of each file should have these words:

> This work is free and unencumbered software released into the public domain. In jurisdictions that recognize copyright laws, the author or authors of this software dedicate any and all copyright interest in the software to the public domain.

- [X] I dedicate any and all copyright interest in this software to the public domain. I make this dedication for the benefit of the public at large and to the detriment of my heirs and successors. I intend this dedication to be an overt act of relinquishment in perpetuity of all present and future rights to this software under copyright law.

### CODE REVIEW

**Comment by paarongiroux** [comment]
Did you mean to merge this PR into dev (rather than the appdata branch?)

File: isis/src/lro/apps/lronac2isis/LroNacArchive.trn [hunk 1]
```diff
@@ -0,0 +1,250 @@
+#
```
Comment (kaitlyndlee) [inline isis/src/lro/apps/lronac2isis/LroNacArchive.trn]: Do you mean to have LRO stuff in here?

File: isis/appdata/serialnumbers/vikSerialNumber0003.trn [hunk 2]
```diff
@@ -0,0 +1,33 @@
+Group = Keyword1
```
Comment (kaitlyndlee) [inline isis/appdata/serialnumbers/vikSerialNumber0003.trn]: Upper camel case and should it be renamed to contain the full mission name?

File: isis/src/lro/apps/lronac2isis/LroNacArchive.trn (hunk 1 above)
Comment (scsides) [reply isis/src/lro/apps/lronac2isis/LroNacArchive.trn]: No, this one slipped into this PR some how. It needs to be in the branch, so I'll leave it for now.

Comment (scsides) [reply isis/src/lro/apps/lronac2isis/LroNacArchive.trn]: No, I need to remove the LRO files

File: isis/appdata/serialnumbers/vikSerialNumber0003.trn (hunk 2 above)
Comment (scsides) [reply isis/appdata/serialnumbers/vikSerialNumber0003.trn]: done in new PR

File: isis/appdata/serialnumbers/Viking1CameraASerialNumber.trn [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+End_Group
+
+Group = Keyword2
+  Auto
+  InputKey       = InstrumentId
+  InputGroup     = "IsisCube,Instrument"
+  InputPosition  = (IsisCube, Instrument)
+  OutputName     = Keyword2
+  OutputPosition = (Group, SerialNumberKeywords)
+  Translation    = (VISB, VISUAL_IMAGING_SUBSYSTEM_CAMERA_B)
```
Comment (krlberry) [inline isis/appdata/serialnumbers/Viking1CameraASerialNumber.trn]: Does "B" need to be in here if the file is just for "A" ?

File: isis/appdata/serialnumbers/Viking1CameraBSerialNumber.trn [hunk 4]
```diff
@@ -0,0 +1,33 @@
...
+
+Group = Keyword2
+  Auto
+  InputKey       = InstrumentId
+  InputGroup     = "IsisCube,Instrument"
+  InputPosition  = (IsisCube, Instrument)
+  OutputName     = Keyword2
+  OutputPosition = (Group, SerialNumberKeywords)
+  Translation    = (VISB, VISUAL_IMAGING_SUBSYSTEM_CAMERA_B)
+  Translation    = (VISA, VISUAL_IMAGING_SUBSYSTEM_CAMERA_A)
```
Comment (krlberry) [inline isis/appdata/serialnumbers/Viking1CameraBSerialNumber.trn]: Does A need to be in here if the file is just for B?

File: isis/appdata/serialnumbers/Viking2CameraASerialNumber.trn [hunk 5]
```diff
@@ -0,0 +1,33 @@
...
+End_Group
+
+Group = Keyword2
+  Auto
+  InputKey       = InstrumentId
+  InputGroup     = "IsisCube,Instrument"
+  InputPosition  = (IsisCube, Instrument)
+  OutputName     = Keyword2
+  OutputPosition = (Group, SerialNumberKeywords)
+  Translation    = (VISB, VISUAL_IMAGING_SUBSYSTEM_CAMERA_B)
```
Comment (krlberry) [inline isis/appdata/serialnumbers/Viking2CameraASerialNumber.trn]: Same as above.

File: isis/appdata/serialnumbers/Viking2CameraBSerialNumber.trn [hunk 6]
```diff
@@ -0,0 +1,33 @@
...
+Group = Keyword2
+  Auto
+  InputKey       = InstrumentId
+  InputGroup     = "IsisCube,Instrument"
+  InputPosition  = (IsisCube, Instrument)
+  OutputName     = Keyword2
+  OutputPosition = (Group, SerialNumberKeywords)
+  Translation    = (VISB, VISUAL_IMAGING_SUBSYSTEM_CAMERA_B)
+  Translation    = (VISA, VISUAL_IMAGING_SUBSYSTEM_CAMERA_A)
+  Translation    = (*, *)
```
Comment (krlberry) [inline isis/appdata/serialnumbers/Viking2CameraBSerialNumber.trn:20]: Same as above.

**Review by krlberry — COMMENTED** [review]
Some questions:

File: isis/appdata/serialnumbers/Viking1CameraASerialNumber.trn (hunk 3 above)
Comment (scsides) [reply isis/appdata/serialnumbers/Viking1CameraASerialNumber.trn]: Since I now have a file for each instrument, no they don't I'll update

**Review by krlberry — APPROVED** [review]

**Pull request merged**